Makefile

```diff
@@ -30,7 +33,16 @@ clean-test-leftovers: ## Cleans up any lingering resources in case tests fail ma
 		&>/dev/null || true
 
 .PHONY: test
-test: install-test-deps test-charts test-examples ## Run all chart tests and example tests
+test: unit-test deployment-test ## Run all unit tests and deployment tests
+
+.PHONY: unit-test
+unit-test: README.md ## Run local development tests that must pass prior to submitting a Pull Request
+
+README.md: $(README_SOURCES)
```

Comment:
Workaround for using industry standard tooling for doc generation in Helm projects!